proto/rill/runtime/v1/queries.proto

```diff
@@ -1292,3 +1300,19 @@ message TableRowsRequest {
 message TableRowsResponse {
   repeated google.protobuf.Struct data = 1;
 }
+
+message ResolveTemplatedStringRequest {
+  string instance_id = 1;
+  // Any string with Go templating
+  string body = 2;
+  // If true, output format tokens instead of raw values.
+  bool use_format_tokens = 3;
+  // Additional filters to inject into metrics SQL queries by metrics view name
+  map<string, Expression> additional_where_by_metrics_view = 4;
+  // Additional time range to add to metrics SQL queries
+  Expression additional_time_range = 5;
```

Comment:
This should use the `TimeRange` type for consistency with the metrics API. The `TimeRange` type has additional features. If the caller doesn't need those features, they already have the ability to pass time expressions through `additional_where_by_metrics_view`.